frameworks/Java/beyondj/setup.sh

```diff
@@ -0,0 +1,16 @@
+WORKING_DIR=beyondj-launcher/deploy
+#FILE=beyondj-launcher/deploy/beyondj.jar
```

Comment:
For the sake of keeping the filebase clean and small, please remove commented lines.